src/shell-init.ts

```diff
@@ -37,12 +37,6 @@ const removeMambaInitBlockFromBashProfile = () => {
   })
 }
 
-const copyMambaBatToMicromambaBat = (options: Options) => {
```

Comment:
We need to keep it so that setup-micromamba can work with micromamba 2.0.0, 2.0.1 and 2.0.2